tldw_Server_API/app/core/Evaluations/benchmark_loaders.py

```
@@ -123,15 +132,19 @@ def stream_large_file(source: str, format: str = 'jsonl',
+                try:
+                    for line in r.text.splitlines():
+                        if line:
+                            chunk.append(json.loads(line))
+                            if len(chunk) >= chunk_size:
+                                yield chunk
+                                chunk = []
+                finally:
+                    try:
+                        r.close()
+                    except Exception:
+                        pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)